### ISSUE TITLE
Allow to show close, minimize buttons in fullscreen mode

### ISSUE THREAD

**Issue by ssoaora** [microsoft/vscode#232937]
Type: <b>Bug</b>

Dear vscode team,

I hope this message finds you well.

Currently I'm not able to see the three buttoons for the vscode workspace that are opened in fullscreen mode on macOS.
Please see the attached pics:
![Image](https://github.com/user-attachments/assets/231b5814-be9f-4ab5-a379-811d2b20b027)

And the 3 buttons that I am referring to:
![Image](https://github.com/user-attachments/assets/371f1e2f-d013-4991-a9bc-c50a701567ea)

I really hope this gets fixed.

Best regards,
Dongmin Lee

VS Code version: Code 1.95.1 (Universal) (65edc4939843c90c34d61f4ce11704f09d3e5cb6, 2024-10-31T05:14:54.222Z)
OS version: Darwin arm64 24.1.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M3 Pro (11 x 2400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|2, 2, 3|
|Memory (System)|18.00GB (3.24GB free)|
|Process Argv|--crash-reporter-id 6ce5b2a3-c772-4c9c-930d-5587ec7b684f --crash-reporter-id 6ce5b2a3-c772-4c9c-930d-5587ec7b684f|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (31)</summary>

Extension|Author (truncated)|Version
---|---|---
js-auto-backticks|cha|1.2.0
vscode-markdownlint|Dav|0.56.0
vscode-eslint|dba|3.0.10
gitlens|eam|15.6.2
vscode-html-css|ecm|2.0.10
vsc-material-theme|Equ|34.7.7
prettier-vscode|esb|11.0.0
auto-close-tag|for|0.5.15
copilot|Git|1.243.0
copilot-chat|Git|0.22.1
vscode-pull-request-github|Git|0.100.0
vsc-python-indent|Kev|1.18.0
vscode-gutter-preview|kis|0.31.2
vscode-language-pack-ko|MS-|1.95.2024103009
debugpy|ms-|2024.12.0
python|ms-|2024.18.0
vscode-pylance|ms-|2024.10.1
jupyter|ms-|2024.10.0
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.0.21
vscode-jupyter-cell-tags|ms-|0.1.9
vscode-jupyter-slideshow|ms-|0.1.6
live-server|ms-|0.4.15
indent-rainbow|ode|8.3.1
vscode-gitignore-generator|pio|1.0.3
material-icon-theme|PKi|5.12.0
gitmoji-vscode|sea|1.2.5
pdf|tom|1.2.2
intellicode-api-usage-examples|Vis|0.2.9
vscodeintellicode|Vis|1.3.2
JavaScriptSnippets|xab|1.8.0

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805cf:30301675
binariesv615:30325510
vsaa593:30376534
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
pythonnoceb:30805159
asynctok:30898717
pythonmypyd1:30879173
2e7ec940:31000449
pythontbext0:30879054
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
01bff139:31013167
dvdeprecation:31068756
dwnewjupytercf:31046870
impr_priority:31102340
nativerepl1:31139838
refactort:31108082
pythonrstrctxt:31112756
cf971741:31144450
iacca1:31171482
notype1:31157159
5fd0e150:31155592
dwcopilot:31170013

```

</details>

**Comment by bpasero** [comment]
@ssoaora a window that is in macOS fullscreen mode typically does not have these three buttons, did you find an app otherwise that does this? The only exception is that if you move your mouse up outside of the window, it should appear:

![Image](https://github.com/user-attachments/assets/391d06a7-01d4-4195-a1f4-76ca3ca634d4)

**Comment by ssoaora** [comment]
> [@ssoaora](https://github.com/ssoaora) a window that is in macOS fullscreen mode typically does not have these three buttons, did you find an app otherwise that does this? The only exception is that if you move your mouse up outside of the window, it should appear:
> 
> ![Image](https://github.com/user-attachments/assets/391d06a7-01d4-4195-a1f4-76ca3ca634d4)

@bpasero Yes the Jetbrains IDEs support the feature I described.
screenshot from CLion:

**Comment by bpasero** [comment]
@ssoaora but are you certain the window is macOS native fullscreen in that case or just maximised? When you click on the green icon do you to maximise further?

![Image](https://github.com/user-attachments/assets/49e76027-2c29-407d-ad60-5d98373ba90b)

**Comment by ssoaora** [comment]
@bpasero Yep if I hover over with my mouse the message shows 'Exit fullscreen mode' like in the screenshot.
![image](https://github.com/user-attachments/assets/4d8bfaf5-24b5-43dd-8658-65e723ed6c05)

**Comment by bpasero** [comment]
Ok interesting, must be a window mode I was not aware of. Treating as feature request.
